README.md

```diff
@@ -4,8 +4,4 @@
 
 This repository builds and publishes the following images:
 
- - atk4/image - build image used in our workflows. Will have tags 7.2, 7.3, 8.0 etc
-
-## Running Locally
-
-Run `make build` to regenerate Dockerfiles and to rebuild all docker images. Check "Makefile" for more details
```

Comment:
if you are replacing make with "make.php" could you mention that in the README instead?

Reply:
done